plugins/modules/dcnm_policy.py

```diff
@@ -637,12 +716,20 @@ def dcnm_policy_get_have(self):
             pl
             for pl in plist
             for wp in self.want
-            if (pl["templateName"] == wp["templateName"])
+            if pl.get("source", "") == ""
```

Comment:
What is this check for? Not clear. Can you please add a comment for pl.get("source", "")?